bundles/org.openhab.binding.bluetooth.daikinmadoka/src/main/java/org/openhab/binding/bluetooth/daikinmadoka/internal/model/commands/GetIndoorOutoorTemperatures.java

```diff
@@ -0,0 +1,88 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.bluetooth.daikinmadoka.internal.model.commands;
+
+import java.util.concurrent.Executor;
+
+import org.eclipse.smarthome.core.library.types.DecimalType;
+import org.openhab.binding.bluetooth.daikinmadoka.internal.model.MadokaMessage;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ *
+ * @author blafois
+ *
+ */
+public class GetIndoorOutoorTemperatures extends BRC1HCommand {
+
+    private final Logger logger = LoggerFactory.getLogger(GetIndoorOutoorTemperatures.class);
+
+    private DecimalType indoorTemperature;
+    private DecimalType outdoorTemperature;
+
+    @Override
+    public byte[] getRequest() {
+        return MadokaMessage.createRequest(this);
+    }
```

Comment:
Since a lot of the commands that use this as their implementation perhaps you should make this the default implementation in `BRC1HCommand`.